components/eam/src/physics/p3/scream/micro_p3.F90

```diff
@@ -1297,6 +1301,7 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_atm,qv,dt,qi,qm,ni,bm,   &
     ze_rain   = 1.e-22_rtype
     diag_eff_radius_qc = 10.e-6_rtype ! default value
     diag_eff_radius_qi = 25.e-6_rtype ! default value
+    diag_eff_radius_qr = 500.e-6_rtype ! default value
```

Comment:
@PeterCaldwell , and the default value

Reply:
To be clear, I chose this fairly arbitrarily, as a value that's kind of agnostic between the category being drizzle or true rain. (I think we should consider using an alternate approach that will get rid of these defaults entirely, but that's a conversation for next week I think.)